tts-python-evaluation-QA.ipynb

```diff
@@ -0,0 +1,455 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "fe74c6db",
+   "metadata": {},
+   "source": [
+    "# TTS Evaluation QA"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1eebb0ba",
+   "metadata": {},
+   "source": [
+    "The purpose of this notebook is to find out any inconsistent audio, transcript pair by comparing ASR generated transcripts of the audios to the original transcripts."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "da0f5b95",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Enter a valid manifest path: /home/siddhartht/tts/tutorials/6097_5_mins/manifest.json\n"
+     ]
+    }
+   ],
+   "source": [
+    "manifest_file = input(\"Enter a valid manifest path: \")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "id": "34c3ab8b",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Enter a valid filepath to save asr predictions: /home/siddhartht/tts/tutorials/6097_5_mins/asr_pred.json\n"
+     ]
+    }
+   ],
+   "source": [
+    "asr_pred = input(\"Enter a valid filepath to save asr predictions: \")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "f658b9eb",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{\"audio_filepath\": \"/home/siddhartht/tts/tutorials/6097_5_mins/audio/presentpictureofnsw_02_mann_0532.wav\", \"text\": \"not to stop more than ten minutes by the way\", \"duration\": 2.6, \"text_no_preprocessing\": \"not to stop more than ten minutes by the way,\", \"text_normalized\": \"not to stop more than ten minutes by the way,\"}\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "!head -n 1 {manifest_file}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "32f33cbb",
+   "metadata": {},
+   "source": [
+    "### Synthesize text from the asr."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "3f5002b0",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "fatal: destination path 'NeMo' already exists and is not an empty directory.\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "## Clone the latest NeMo.\n",
+    "! cd ~ && git clone https://github.com/NVIDIA/NeMo.git"
```

Comment:
Please pip install NeMo and just wget/curl (preferably via python) the example `transcribe_speech.py` file.